packages/pn-pa-webapp/src/pages/__test__/ApiKeys.page.test.tsx

```diff
@@ -1,14 +1,21 @@
 /*
-  Scrivere TEST opportuni non appena si avrà il quadro completo su FE/BE
-  Issue di riferimento: PN-1845
-
-  Ci sono un paio di test dove si è provveduti a skipparli per evitare il fallimento dei test di tutta la webapp
+  Remove all skip() and comments until a weird issue is solved:
+
+  --- THE ISSUE ---
+  TypeError: Cannot read properties of undefined (reading 'fulfilled')
```

Comment:
In my opinion the issueis not weird at all. Since you are mocking `getApiKeys` with a plain function, the mocked value won't include a `fulfilled` attribute. Moreover, since also `dispatch` is being mocked, in fact `getApiKeys`, either real or mocked, won't be invoked.
Hence, I think that if you mock `dispatch`, then you lose the ability of checking whether `getApiKeys` is invoked. If you "renounce" to such check, then you can avoid mocking `getApiKeys`, this suffices to make the tests work (I just tried).